common/static/common/js/utils/edx.utils.validate.js

```diff
@@ -22,6 +22,7 @@
                     validate: {
 
                         template: _.template('<li><%- content %></li>'),
```

Comment:
Maybe we could just pass another argument `className` along with `content`:
```suggestion
                        template: _.template('<li <%- suppressAttr %>><%- content %></li>'),
```